connect/api/src/main/java/org/apache/kafka/connect/data/Values.java

```diff
@@ -1059,6 +1085,22 @@ public String next() {
             return previousToken;
         }
 
+        public String nextTokens(int n) {
```

Comment:
Maybe just call it `next(int)` to match existing `next()` method?

Reply:
Changed.